irohad/ametsuchi/impl/postgres_wsv_common.hpp

```diff
@@ -28,11 +29,33 @@ namespace iroha {
      * Return function which can execute SQL statements on provided transaction
      * @param transaction on which to apply statement.
      * @param logger is used to report an error.
+     * @return Result with pqxx::result in value case, or exception message
+     * if exception was caught
+     */
+    inline auto makeExecuteResult(pqxx::nontransaction &transaction,
+                                  logger::Logger &logger) noexcept {
```

Comment:
Maybe make it as a class function so there's no need in passing logger?

Reply:
It will be used by wsv query in the future, so no point in making it exclusive to wsv command